src/mame/act/victor9k.cpp

```diff
@@ -829,6 +856,19 @@ void victor9k_state::victor9k(machine_config &config)
 	m_fdc->syn_wr_callback().set(I8259A_TAG, FUNC(pic8259_device::ir0_w)).invert();
 	m_fdc->lbrdy_wr_callback().set_inputline(I8088_TAG, INPUT_LINE_TEST).invert();
 
+	NSCSI_BUS(config, m_scsibus);
+	NSCSI_CONNECTOR(config, "scsi:0", scsi_devices, "harddisk", false);
+	NSCSI_CONNECTOR(config, "scsi:7").option_set("v9kdmaib", V9KDMAIB).machine_config(
+		[this](device_t *device)
+		{
+			v9kdmaib_device &v9kdmaib(downcast<v9kdmaib_device &>(*device));
+
+			device->set_clock(15_MHz_XTAL / 3);
+			v9kdmaib.irq_handler().append(m_pic, FUNC(pic8259_device::ir4_w));
+			v9kdmaib.dma_read().set(*this, FUNC(victor9k_state::hd_dma_r));
+			v9kdmaib.dma_write().set(*this, FUNC(victor9k_state::hd_dma_w));
+		});
```

Comment:
Why not allow more SCSI devices?  Did the system not physically allow it somehow?